training/distributed_training/pytorch/model_parallel/gpt-j/fp16/__init__.py

```diff
@@ -1,6 +1,5 @@
 # coding=utf-8
-# Copyright (c) 2022, NVIDIA CORPORATION.  All rights reserved.
-# Modifications Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Keep the Amazon modifications copyright?

Reply:
Included proper copyright.